Dockerfile

```diff
@@ -22,11 +22,22 @@ COPY --from=jenkinsfilerunner-mvncache /mavenrepo /mavenrepo
 ADD . /jenkinsfile-runner
 RUN cd /jenkinsfile-runner && mvn package
 
-FROM jenkins/jenkins:${JENKINS_VERSION}
+FROM jenkins/jenkins:${JENKINS_VERSION} as jenkins
 USER root
-RUN mkdir /app && unzip /usr/share/jenkins/jenkins.war -d /app/jenkins
+# Delete big files not needed
+RUN mkdir /app && unzip /usr/share/jenkins/jenkins.war -d /app/jenkins && \
+  rm -rf /app/jenkins/scripts /app/jenkins/jsbundles /app/jenkins/css /app/jenkins/images /app/jenkins/help /app/jenkins/WEB-INF/detached-plugins /app/jenkins/winstone.jar /app/jenkins/WEB-INF/jenkins-cli.jar /app/jenkins/WEB-INF/lib/jna-4.5.2.jar
+
+FROM openjdk:8-jdk
+ENV JENKINS_UC https://updates.jenkins.io
+USER root
+RUN mkdir -p /app /usr/share/jenkins/ref/plugins
+COPY --from=jenkins /app/jenkins /app/jenkins
+COPY --from=jenkins /usr/local/bin/install-plugins.sh /usr/local/bin/install-plugins.sh
+COPY --from=jenkins /usr/local/bin/jenkins-support /usr/local/bin/jenkins-support
 COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
-RUN /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
+# empty war file to workaround https://github.com/jenkinsci/docker/pull/786
+RUN echo UEsFBgAAAAAAAAAAAAAAAAAAAAAAAA== | base64 -d > /usr/share/jenkins/jenkins.war && /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt && rm -f /usr/share/jenkins/jenkins.war
```

Comment:
Could we just integrate a release of that?

Reply:
that was just merged, can use in the next release